Client/src/Utils/Theme/lightTheme.js

```diff
@@ -12,6 +12,14 @@ const background = {
 	fill: "#F4F4F4",
 	accent: "#f9fafb",
 };
+
+export const gaugeHeaderCommon = {
+	fontWeight: 400,
+	fill: text.tertiary,
+	textAnchor: "middle",
+	dominantBaseline: "middle",
+}
+
```

Comment:
Not sure if it is a good idea to export these from the theme. Maybe set this on the component, and the text.tertiary can come from the theme

Reply:
This export is for reuse with darktheme

Reply:
Why don't we put this information, that just makes sense for this component, inside the component?

Reply:
Resolved in : https://github.com/bluewave-labs/bluewave-uptime/pull/1087/commits/94e4d1b1cb9497d56bca5c59a3734595bffb57bd